src/irmin/store.ml

```diff
@@ -331,6 +331,12 @@ module Make (P : S.PRIVATE) = struct
         (function
           | Import_error e -> Lwt.return (Error (`Msg e))
           | e -> Fmt.kstrf Lwt.fail_invalid_arg "impot error: %a" Fmt.exn e)
+
+    let iter_nodes t ~min ~max ~node ~contents ~skip () =
+      Graph.iter (graph_t t) ~min ~max ~node ~contents ~skip ()
```

Comment:
Out of interest, is it actually necessary to pass all of these arguments or can they be eta-reduced? Either way is fine.

Reply:
It's because I changed some optional arguments from `Graph.iter` to named ones in `iter_nodes`, which is not very clean anyway. Changed them back to optional, the two functions have now similar arguments.